straxen/plugins/events/_event_s2_positions_base.py

```diff
@@ -17,11 +17,10 @@ class EventS2PositionBase(strax.Plugin):
 
     algorithm: Optional[str] = None
     compressor = "zstd"
-    parallel = True
 
     min_reconstruction_area = straxen.URLConfig(
```

Comment:
The `min_reconstruction_area` is changed to 0 because I want to run pos-rec for `peaklets` regardlessly.